addons/website/static/src/snippets/s_searchbar/000.js

```diff
@@ -164,6 +164,9 @@ publicWidget.registry.searchBar = publicWidget.Widget.extend({
                 widget: this,
             }));
             this.$menu.css('min-width', this.autocompleteMinWidth);
+            for (const el of this.$menu[0].children) {
+                el.onkeydown = this._onKeydown.bind(this);
+            }
```

Comment:
Can't you delegate the keydown event instead?

Something like changing
```javascript
'keydown .search-query': '_onKeydown',
```
into
```javascript
'keydown .search-query, .dropdown-item': '_onKeydown',
```